### ISSUE TITLE
data sets pages

### ISSUE THREAD

**Issue by jingchunzhu** [ucscXena/ucsc-xena-client#153]
currently the "data sets pages" tool on xena browser only report "loading error" when the genomics data sets itself has an error, but it does not check if the associated "probeMap" loading also generated errors. This has caused a few problems for users that they thought their data is properly loaded to xena hub, while there is in fact a formatting problem with the probeMap file, which leads to failure of visualization of the data on the xena browser. 

check loading error on the associated probeMap file, and reflect that on "data sets pages"

**Comment by jingchunzhu** [comment]
code fix in xena_datapages module at https://github.com/ucscXena/xena_datapages

**Comment by souravsingh** [comment]
@jingchunzhu Is this the fix for the issue? https://github.com/ucscXena/xena_datapages/blob/master/js/datapages.js#L109

**Comment by acthp** [comment]
@souravsingh No, that's not a fix. The bug is that some datasets have associated probemaps, and the probemap load can fail. We currently don't report that information, or allow the user to delete the failed probemap.

The fix would be to see if the dataset has a probemap, via this var:

https://github.com/ucscXena/xena_datapages/blob/master/js/datapages.js#L733

and check its status. If the status is 'error', render this on the page for the user.

Additionally, we might want a 'delete' button for the probemap, but there are complications around this that we should investigate first. E.g. the probemap might be used by other datasets, so deleting it would affect them. Also, deleting the probemap won't remove the probemap name from the dataset metadata, so queries against the dataset may fail because of missing probemap.

In any case, the first step is to just check probemap status and report it to the user.
